index.html

```diff
@@ -67,7 +67,8 @@
     </nav>
 
     <header class="bg-colorsplash">
-      <button class="subscribe"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button>
+      <!--button class="subscribe"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button-->
+      <button class="subscribe btn-lg" data-toggle="modal" data-target="#mc_embed_signup">Subscribe to our mailing list here!</button>
```

Comment:
@howmanywhat Here's another comment :-)

Reply:
ok, i'll fix that and do another pull request? Can you take a quick look at the button style changes from a browser to see if it's ok? (i can change them back at the same time if the current appearance more desirable)

Reply:
o wait, just more commits to this branch github says below :)